ci/glci/model.py

```diff
@@ -11,7 +11,8 @@
 import yaml
 
 own_dir = os.path.abspath(os.path.dirname(__file__))
-repo_root = os.path.abspath(os.path.join(own_dir, os.path.pardir, os.path.pardir))
+repo_root = os.path.abspath(os.path.join(
+    own_dir, os.path.pardir, os.path.pardir))
```

Comment:
nit: place closing paranthesis in new line? this is not an "official" recommendation as of PEP8, but would be consistent with the other code from `ci`